packages/docs/concepts/lowdefy-schema.yaml

```diff
@@ -24,204 +24,262 @@ _ref:
-                      icon: AiOutlineControl
-            ```
+            [TODO]: <> (# TODO: Add plugins things here)
+
+            ## Menus
+
+            Menu objects are objects with links to pages. They are filtered to only show pages that the user is authorized to see and are used by blocks such as `PageSiderMenu` to render the links in the menu. If no menus are provided, a default menu is created, with links to all of the defined pages, and with their pageIds as menu item titles.
+
+            See more about how menu objects are used and defined [here](/menus).
 
-            # Pages
+            ## Pages
 
             Pages in a Lowdefy app are actually just blocks, the building blocks of a Lowdefy app, with a few extra restrictions and features.
 
             Each page should have an `id` that is unique among all pages in the app. Each page is served with the `pageId` as the url route. That is, if you create a page with id `page1`, it will be served at `domain.com/page1`.
 
-            Page blocks should be of category `context`, in order to create a context for the page. This means they always have the context initialization events like `onInit` to initialize the page.
+            [TODO]: <> (# TODO: Below will change once we add the head configuration option on pages. Discuss that here.)
 
             If `properties.title` is set on a page block, the title will be set as the page title (This is the title displayed on the tabs in your browser).
 
-            # References and templates
-
-            References are used to split the configuration of an app into logically distinct files, and to reuse configuration in the app. References can be used almost anywhere in the configuration, as long as the configuration remains valid YAML.
-
-            References are made using the [`_ref`](/_ref) operator. If the referenced file has a `.yaml` or `.json` extension, the contents of the file will be parsed, else the file content is included as a string (this is useful for `.md` or `.html` files). As an example of splitting an app into logically distinct files, references can be used to write each page as a separate file:
+            Let's have a look at how to define a page and it's blocks. We can start with a simple page a card block on it. We can add a title block and a paragraph block to our card so that it won't be so empty.
 
             ###### lowdefy.yaml
             ```yaml
             lowdefy: LOWDEFY_VERSION
 
             pages:
-              _ref: pages/page1.yaml # Path to the referenced file. Always from the root of the project.
-              _ref: pages/page1.yaml
+              - id: page1
+                type: PageHeaderMenu
+                properties:
+                  title: Page 1
+                blocks:
+                  - id: content_card
+                    type: Card
+                    blocks:
+                      - id: title
+                        type: Title
```

Comment:
Just thinking now, that since a card has a title property is maybe a design anti-pattern to put a title inside a card like here.. Perhaps just use a paragraph and a button.

Reply:
Change has been made in #1314.